marklogic-data-hub/src/main/resources/ml-modules/services/tracing.xqy

```diff
@@ -23,6 +23,9 @@ import module namespace debug = "http://marklogic.com/data-hub/debug-lib"
 import module namespace trace = "http://marklogic.com/data-hub/trace"
   at "/com.marklogic.hub/lib/trace-lib.xqy";
 
+import module namespace perflog = "http://marklogic.com/data-hub/perflog-lib"
```

Comment:
This module doesn't need performance tracing either. Please back it out of the PR.